### PR TITLE
Add access controls

### PR DESCRIPTION
**Pull request by squirrelo**
This breaks apart functionality in labadmin into discrete access groups that must be set per user.

Requires [american gut #587](https://github.com/biocore/american-gut-web/pull/587)

### CODE REVIEW

File: knimin/handlers/ag_add_barcode_kit.py [hunk 1]
```diff
@@ -7,11 +7,13 @@
 class AGAddBarcodeKitHandler(BaseHandler):
```
Comment (wasade) [inline knimin/handlers/ag_add_barcode_kit.py]: would it be possible to have the methods default to a high access level? In other words, unless explicitly stated, you need `Admin`. It's conservative in the event some handlers get over looked, or future development forgets to add the `self.has_access` calls

Comment (squirrelo) [inline knimin/handlers/ag_add_barcode_kit.py]: I can add that, although that requires adding a base access level to the database as well.

Comment (wasade) [inline knimin/handlers/ag_add_barcode_kit.py]: So then general forms would have something like `self.has_access('basic')`?

Comment (squirrelo) [inline knimin/handlers/ag_add_barcode_kit.py]: yup

**Comment by squirrelo** [comment]
This is ready for review, although I can't add tests until the AG DB PR is in. Also, acronyms.

**Comment by squirrelo** [comment]
ping @EmbrietteH @wasade

**Comment by wasade** [comment]
seems good

**Pull request merged**